subprojects/core/src/main/java/org/gradle/invocation/DefaultGradle.java

```diff
@@ -80,7 +81,7 @@ public DefaultGradle(GradleInternal parent, StartParameter startParameter, Servi
         buildListenerBroadcast.add(new BuildAdapter() {
             @Override
             public void projectsLoaded(Gradle gradle) {
-                rootProjectActions.execute(rootProject);
+                getDomainObjectConfigurator().configure(rootProject, rootProjectActions, "initialization scripts");
```

Comment:
This should be a simple build operation called `executing rootProject {} callbacks`.